src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -26,7 +26,7 @@
 
     public ShooterVisionSubsystem goalVisionSubsystem;
 
-    public IndexSubsystem indexSubsystem;
+    public static IndexSubsystem indexSubsystem;
 
```

Comment:
Shouldn't need to be `static`. If something needs it to be static, that's a bug in that code...